user-access-service/src/test/java/no/unit/nva/database/DatabaseServiceImplTest.java

```diff
@@ -1,108 +0,0 @@
-package no.unit.nva.database;
-
-import static no.unit.nva.database.RoleService.ROLE_NOT_FOUND_MESSAGE;
-import static nva.commons.core.attempt.Try.attempt;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.containsString;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.mockito.ArgumentMatchers.anyString;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
-import com.amazonaws.services.dynamodbv2.document.Table;
-import com.amazonaws.services.dynamodbv2.model.AmazonDynamoDBException;
-import no.unit.nva.useraccessmanagement.dao.DynamoEntryWithRangeKey;
-import no.unit.nva.useraccessmanagement.dao.RoleDb;
-import no.unit.nva.useraccessmanagement.dao.UserDb;
-import no.unit.nva.useraccessmanagement.exceptions.InvalidEntryInternalException;
-import no.unit.nva.useraccessmanagement.model.RoleDto;
-import no.unit.nva.useraccessmanagement.model.UserDto;
-import nva.commons.logutils.LogUtils;
-import nva.commons.logutils.TestAppender;
-import org.hamcrest.core.StringContains;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.function.Executable;
-
-public class DatabaseServiceImplTest extends DatabaseAccessor {
-
-    public static final String SOME_INSTITUTION = "someInstitution";
-    public static final String SOME_USERNAME = "someUsername";
-    public static final String EXPECTED_EXCEPTION_MESSAGE = "ExpectedExceptionMessage";
-
-    private UserDto someUser;
-    private IdentityServiceImpl databaseService;
-
-    @BeforeEach
-    public void init() throws InvalidEntryInternalException {
-
-        someUser = UserDto.newBuilder().withUsername(SOME_USERNAME).build();
-        databaseService = new IdentityServiceImpl(initializeTestDatabase(), envWithTableName);
-    }
-
-    @Test
-    public void getRoleExceptionWhenItReceivesInvalidRoleFromDatabase()
-        throws InvalidEntryInternalException {
-
-        IdentityService serviceThrowingException = mockServiceThrowsExceptionWhenLoadingRole();
-        RoleDto sampleRole = EntityUtils.createRole(EntityUtils.SOME_ROLENAME);
-        Executable action = () -> serviceThrowingException.getRole(sampleRole);
-        RuntimeException exception = assertThrows(RuntimeException.class, action);
-
-        assertThat(exception.getMessage(), containsString(EXPECTED_EXCEPTION_MESSAGE));
-    }
-
-    @Test
-    public void getRoleThrowsInvalidEntryInternalExceptionWhenItReceivesInvalidRoleFromDatabase()
-        throws InvalidEntryInternalException {
-
-        IdentityService service = mockServiceReceivingInvalidRoleDbInstance();
```

Comment:
This test is impossible to happen, because the role name is part of the key. The test has been removed.